src/pages/BookAppointment.js

```diff
@@ -0,0 +1,6 @@
+function BookAppointment() {
+  return (
+    <h2>This is BookAppointment page</h2>
+  );
+}
+export default BookAppointment;
```

Comment:
Also it is always a good practice to use (.jsx) extension for react components and this helps to tell that your component contains both javascript and HTML code